vpr/src/timing/timing_graph_builder.cpp

```diff
@@ -273,56 +518,56 @@ void TimingGraphBuilder::add_block_to_timing_graph(const AtomBlockId blk) {
         //and between the internal SOURCEs and SINKS for sequential blocks
```

Comment:
Did I missing something, or is there no loop that constructs sequential to sequential internal delays?